deeppavlov/core/agent_v2/state_schema.py

```diff
@@ -73,3 +101,19 @@ def to_dict(self):
             'bot': [u.to_dict() for u in self.users if u.user_type == 'bot'][0],
             'channel_type': self.channel_type
         }
+
+
+class SkillResponse(DynamicDocument):
+    """
+    Data Model for Skills responses. It specifies behavior hypothesis from particular skill
+    after processing new Utternace
+    """
+    text = StringField()
+    confidence = FloatField()
```

Comment:
I think, we could set default value to 1.0 here.

Reply:
fixed